Makefile

```diff
@@ -132,3 +132,12 @@ build-rpms:
 build-images:
 	hack/build-images.sh
 .PHONY: build-images
+
+# Generate CRD OpenAPIv3 validation 
+update-codegen-crds:
+	go run ./vendor/github.com/openshift/library-go/cmd/crd-schema-gen/main.go --apis-dir vendor/github.com/openshift/api
+
+verify-codegen-crds:
```

Comment:
I think we should add this to the tests run by Travis.

Reply:
I'm leaning toward moving this (at least) to a separate PR,we don't need this atm